src/sql/workbench/browser/modelComponents/listView.component.html

```diff
@@ -0,0 +1,12 @@
+<div role="listbox" class="listview-container" [ngStyle]="styles" [style.width]="width" [style.height]="height">
+	<div *ngIf="title" class="listview-title">{{title.text}}</div>
+	<ng-container *ngFor="let option of options">
+		<li #optionDiv role="option " [ngClass]="{ 'listview-li': true,  'li-active':isOptionSelected(option.id)}"
```

Comment:
We should be using already-implemented components whenever we can instead of recreating common controls like this : https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/browser/ui/list/listView.ts#L202

Reply:
please take a look at this component, we might be able to re-use this one.